funnel/templates/notifications/comment_report_received_email.html.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "notifications/layout_email.html.jinja2" %}
+{% block content -%}
+
+<p>{% trans actor=view.actor.fullname %}Hi {{ actor }},{% endtrans %}</p>
+
+<p>
+    {% trans url=url_for('siteadmin_review_comment', report=view.report.uuid_b58) -%}
+    A comment has been reported as spam. <a href="{{ url }}">Please review</a>.
+    {%- endtrans %}
+</p>
```

Comment:
This is a use case for adding the actions layer to notifications. I'll spec it out along with the Telegram transport, but we could do a couple quick fixes here:

1. Include a copy of the comment. (This may be a bad idea if it triggers spam detection in the email client.)

2. Use a button bar in the template. `<p class="btn-bar"><a class="btn">`